api/services/healthService.ts

```diff
@@ -0,0 +1,30 @@
+import { makeSCOSRequest } from './scosService';
+import HealthDto from '../models/dto/healthDto';
+import getLogger from '../utils/logger';
+
+const { AGENCIES_TABLE, CATEGORIES_TABLE } = process.env;
```

Comment:
Can you describe why you didn't just use a constant here? Are we going to have a dev table name or something?

I think a better way to do this may be to create a `contants.js` file and we can set up the constants either as strings or from `process.env` but this way we can always import from `constants.js`.

```js
// constants.js
export const AGENCIES_TABLE = process.env.AGENCIES_TABLE || 'default_table_name_here'
export const CATEGORIES_TABLE = process.env.CATEGORIES_TABLE || 'default_table_name_here'
```

Reply:
I dig it 👍